Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -46,6 +46,41 @@ public enum RichTextAction: Identifiable, Equatable {
 
     /// Undo the latest change.
     case undoLatestChange
+    
+    // Change background color
+    case backgroundColor(ColorRepresentable)
+    
+    // Change foreground color
+    case foregroundColor(ColorRepresentable)
```

Comment:
For all these, we should have a single `case setColor(RichTextColor, to: ColorRepresentable)`.